src/app/components/common/node-editor/editor.ts

```diff
@@ -105,16 +105,30 @@ export async function createEditor(container: HTMLElement, injector: Injector, f
       if (context === 'root') {
         return {
           searchBar: false,
-          list: [
-            {
-              label: 'Filter', key: '1', handler: () => {
-                let filter = new Filter(socket, { pluginName: '', enabled: 'false', filterName: 'Filter' });
```

Comment:
Is it safe to remove? 

What was this being used for?

Reply:
It was not is used. It was for providing the add filter context menu on the canvas but we did not add that feature.